### PR TITLE
LibX11: drop unused xlibi18n

### PR DESCRIPTION
**Pull request by uli42**
remove support for _loadable_ `xlibli18n`

As i18n is used by the application and we are the ones controlling the application (nxagent) it might also be possible to drop that i18n stuff completely! I have not investigated further, though.

### CODE REVIEW

File: nx-X11/config/cf/X11.tmpl [hunk 1]
```diff
@@ -126,9 +126,6 @@ XORGRELSTRING = XorgManVersionString
 #define BuildX11Lib		BuildLibraries
 #endif
 
-#ifndef BuildLoadableXlibI18n
-#define BuildLoadableXlibI18n	(HasDlopen && SharedLibX11)
-#endif
```
Comment (sunweaver) [inline nx-X11/config/cf/X11.tmpl:131]: I just grepped for more places where xlibi18n gets mentioned:

```
 [mike@minobo nx-X11 (uli42-pr/remove_xlibi18n)]$ grep -ri xlibi18n *
config/cf/lnxLib.tmpl:#define SharedXlibi18nReqs $(LDPRELIB) $(XONLYLIB)
config/cf/X11.tmpl: * This disables XLOCALEDIR.  Set it if you're using BuildLoadableXlibI18n,
```

Comment (sunweaver) [reply nx-X11/config/cf/X11.tmpl:131]: Please amend those and include in this commit, too.

**Review by sunweaver — CHANGES_REQUESTED** [review]
I found two more occurences of xlibi18n (grep -i). Please handle those via this PR, too.

Except from that, this PR is fine.

File: nx-X11/config/cf/X11.tmpl (hunk 1 above)
Comment (uli42) [reply nx-X11/config/cf/X11.tmpl:131]: rebased and added the two requested changes

**Review by sunweaver — APPROVED** [review]

**Pull request merged**